Alteracoes 2019

essantos97 opened the pull request.

essantos97 committed:
Alterações dos membros ecompJr

Adicionadas as fotos novas dos membros

essantos97 committed:
Alterações no site

Adição dos novos membros da empresa, atualização do portfólio, correção de erros existentes, reformulação das pastas, adição de novas fotos ao portfólio, criação de páginas mais organizadas.

essantos97 committed:
Atualização do site

Alteração de mensagem presente no portfólio da alfajr.

essantos97 committed:
Update Members, Remove Link of the Selection Process

Adição dos membros que ainda faltavam no site da empresa, atualização da página para retirada do link do processo seletivo.

ssscassio commented on assets/js/members-page.js:
```diff
@@ -60,8 +60,17 @@
+          {
+              name: 'Adriel Brito',
+              email: 'adrielbrito@ecompjr.com.br',
+              github: '',
```
Acredito que já possa adicionar o github desses membros.

ssscassio commented on assets/js/members-page.js:
```diff
@@ -60,8 +60,17 @@
+              email: 'adrielbrito@ecompjr.com.br',
+              github: '',
+              job: 'Membro Consultor',
               text: '',
```
Adicionar textos referentes aos membros adicionados.

ssscassio reviewed (changes requested):
Concluindo essas alterações considero aprovado.

Não podemos deixar esse tipo de pull Request passar tanto tempo sem ser aprovado.

essantos97 committed:
Alteração do endereço de site do colegiado de computação.

Foi realizada a alteração do endereço do site do colegiado do curso de engenharia no portfólio.

essantos97 committed:
Merge branch 'master' into Alteracoes-2019
